main/acle.md

```diff
@@ -1019,13 +1003,10 @@ Including `<arm_sve.h>` also includes the following header files:
 
 `<arm_neon_sve_bridge.h>` defines intrinsics for moving data between
 Neon and SVE vector types; see [NEON-SVE Bridge](#neon-sve-bridge)
-for details.  The `__ARM_NEON_SVE_BRIDGE` macro should be tested
-before including the header:
+for details.
 
 ``` c
-  #ifdef __ARM_NEON_SVE_BRIDGE
```

Comment:
I believe this should remain because it's a test if the compiler supports this software extension rather than something related to a target feature.